opus/application/static_media/js/browse.js

```diff
@@ -1155,17 +1162,25 @@ var o_browse = {
             .attr("data-id", opusId);
     },
 
+    getDetailURL: function(opusId) {
+        let tab = opus.getCurrentTab();
+        opus.prefs.detail = opusId;
+        let link = "/opus/#/" +  o_hash.updateHash();
```

Comment:
Do we really have to call updateHash() here, or is there some way to get the current hash that was previously computed? The end of updateHash() sets window.local.hash - can we just use that instead?
Also there's an extra space after the "+"

Reply:
we need to call updateHash because we have to add the detail= part that comes from changing the opus.prefs.

Reply:
modified using hashArray so that the original hash is left untouched.